src/components/tx-flow/flows/UpsertRecovery/UpsertRecoveryFlowIntro.tsx

```diff
@@ -13,9 +13,9 @@ import commonCss from '@/components/tx-flow/common/styles.module.css'
 const RecoverySteps = [
   {
     Icon: RecoveryRecoverers,
-    title: 'Choose a Recoverer and set a delay',
+    title: 'Choose a Recoverer and set a review window',
     subtitle:
-      'Only your chosen Recoverer can initiate the recovery process. The process can be cancelled at any time during the delay period.',
+      'Only your chosen Recoverer can initiate the recovery process. The process can be cancelled at any time during the review window.',
```

Comment:
I think the process can be cancelled at any time. (not just during the review window)

Reply:
As per discussion, this will be left as is because the flow is only seen by the owner and it emphasises that the owner has cancellation rights before the Recoverer.